### DISCUSSION TITLE
Automatically Add Contacts from SimpleLogin to Proton Mail's Contact List

### DISCUSSION THREAD

**Discussion by jeremy-vogel** [simple-login/app#2002] [Feature Requests]
Whenever I create a reverse alias in SimpleLogin, it would be nice if it would automatically be transferred to Proton Mail's contact list as a reverse alias so that if I go to send an email from Proton Mail, it will use the reverse alias without having to either copy and paste from SimpleLogin, or add the reverse alias as a contact in Proton Mail first.  

It would also be nice if I tried to send to a Proton Mail contact and it matched a contact in SimpleLogin, it could alert me and ask if I want to send to the reverse alias instead.

**Comment by winklerrr** [comment]
Any news on this?
My post is related: #2426
